### PR TITLE
{numlib}[intel/2016b] Armadillo 7.950.1 w/ Python 2.7.12

### PR DESCRIPTION
**Pull request by mboisson**
This PR requires merging https://github.com/easybuilders/easybuild-easyblocks/pull/1213 first

### CODE REVIEW

**Comment by mboisson** [comment]
yeah, requires a PR on easyblocks for this PR to work

**Comment by vanzod** [comment]
@mboisson Are you planning to upload a specific easyblock?

**Comment by mboisson** [comment]
There's already one. See the OP

**Comment by boegel** [comment]
broken tests fixed by https://github.com/easybuilders/easybuild-easyconfigs/pull/4849

File: easybuild/easyconfigs/a/Armadillo/Armadillo-7.950.1-intel-2016b-Python-2.7.12.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+homepage = 'http://arma.sourceforge.net/'
+description = """Armadillo is an open-source C++ linear algebra library (matrix maths) aiming towards
+ a good balance between speed and ease of use. Integer, floating point and complex numbers are supported,
+ as well as a subset of trigonometric and statistics functions."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+sources = [SOURCELOWER_TAR_XZ]
+source_urls = ['http://sourceforge.net/projects/arma/files']
```
Comment (boegel) [inline easybuild/easyconfigs/a/Armadillo/Armadillo-7.950.1-intel-2016b-Python-2.7.12.eb]: @mboisson We're slowly working towards making SHA256 checksums a requirement for new easyconfig, so please include it?

```python
checksums = ['sha256checksumgoeshere']
```

Comment (mboisson) [inline easybuild/easyconfigs/a/Armadillo/Armadillo-7.950.1-intel-2016b-Python-2.7.12.eb]: How do I get the correct checksum ?

Comment (boegel) [inline easybuild/easyconfigs/a/Armadillo/Armadillo-7.950.1-intel-2016b-Python-2.7.12.eb]: On Linux: `sha256sum path/to/source`; on macOS: `shasum -a 256 path/to/sum`

We have plans for adding support in EasyBuild itself for this (something like `eb --inject-checksum=sha256 foo.eb`), but it's no more than an idea for now...

Comment (mboisson) [inline easybuild/easyconfigs/a/Armadillo/Armadillo-7.950.1-intel-2016b-Python-2.7.12.eb]: Done

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2451.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/6a0bc147c400161f6e74fc32960793b2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2115.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ef8e2421974c776626d29c3369581f99 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @mboisson!

**Pull request merged**